charts/astronomer/values.yaml

```
@@ -55,7 +58,7 @@ astroUI:
 houston:
   prismaConnectionLimit: 5
   replicas: 2
-  # This only applies when replicas > 3
+  # This only applies when replicas >
```

I assume this deletion was a mistake
```suggestion
  # This only applies when replicas > 3
```